src/btree/bt_vrfy.c

```diff
@@ -280,12 +284,31 @@ __verify_key_hs(WT_SESSION_IMPL *session, WT_ITEM *key, WT_VSTUFF *vs)
 #else
         WT_UNUSED(vs);
 #endif
-    }
 
+        /* Perform Verification of data continuity */
+        if (orig_start.timestamp < stop.timestamp) {
+            ts1_bp = __verify_timestamp_to_pretty_string(stop.timestamp);
+            ts2_bp = __verify_timestamp_to_pretty_string(orig_start.timestamp);
+            WT_ERR_MSG(session, WT_ERROR,
+              "Key %s has a history store stop timestamp %s newer than its start timestamp %s",
```

Comment:
Perhaps something along the lines of: "Key %s has an overlap between a value starting at %s and a value ending at %s" to indicate clearly that they are two different values with bad timestamps. Perhaps even just printing the full start and stop time pairs for both would be nice, since it gives more context into the issue. They are also included when printing the `'page_addr_string'` fyi

Reply:
I definitely see your point about having a more clear value and string of timestamps in the string, but I would also like to conform to original style in **__verify_page_cell**, so I'm thinking we do a compromise of both, and do a string on the lines of "In the Btree %d, page reference at %s, Key %s has a overlap of timestamp ranges between history store stop timestamp %s being newer than a more recent timestamp range having start timestamp %s", let me know what you think of that message

Reply:
Sounds good, maybe simplify the wording a bit so it's more consistent with other messages.